scss/components/_flex.scss

```diff
@@ -17,6 +17,13 @@
     }
   }
 
+  .align-central {
+    align-content: center;
+    @include flex-align($x: 'center');
+    @include flex-align($y: 'middle');
```

Comment:
Also, Please change the internal scss to this below 

```scss
.align-center-middle {
    align-content: center;
    @include flex-align($x: center, $y: middle);
 }
```
Ref. => http://foundation.zurb.com/sites/docs/flexbox.html#helper-mixins